src/main/java/org/web3j/utils/Account.java

```diff
@@ -38,7 +38,7 @@ public static Boolean isAddress(String address) {
 
     private static Boolean validateChecksumAddress(String address) {
         String formattedAddress = address.replace("0x", "").toLowerCase();
```

Comment:
I have to validate eth addresses in a current project - I modified your version:

`String formattedAddress = address.replace("0x", "");
 String lowercaseAddress = formattedAddress.toLowerCase();
 String hash = Numeric.toHexStringNoPrefix(Hash.sha3(lowercaseAddress.getBytes()));
     for (int i = 0; i < 40; i++) {
         if (Character.isLetter(formattedAddress.charAt(i))) {
`